migration.rst

```diff
@@ -316,19 +312,22 @@ somewhat like this::
 
     class LegacyBridge
     {
-        public static function prepareLegacyScript(Request $request, Response $response, string $publicDirectory): ?string
+        public static function handleRequest(Request $request, Response $response, string $publicDirectory): ?string
         {
-            // If Symfony successfully handled the route, you do not have to do anything.
-            if (false === $response->isNotFound()) {
-                return null;
-            }
-
             // Figure out how to map to the needed script file
-            // from the existing application and possibly (re-)set
-            // some env vars.
-            $legacyScriptFilename = ...;
+            // from the existing application.
+            function getScript($requestPathInfo) { ... }
+
+            $p = $request->getPathInfo();
+            $legacyScriptFilename = getScript($p);
```

Comment:
I see what you're saying.  This was actually a mapping function defined in the bridge, not in the legacy app.  I'll add a quick note to the code.

Reply:
I've added quite a few more comments to the LegacyBridge and extracted a function so that the mapping is made clearer, as it's really the core function of the class.  Hopefully it's not too verbose, I tried to keep it concise.

FWIW my own LegacyBridge I used in my (messy messy legacy) project is at https://github.com/jzohrab/lwt/blob/master/src/LegacyBridge.php.  I added a check in mine to ensure the bridge wasn't handling css and js files, because I messed up the Apache rewrites and was getting bad mime types -- wasted hours sorting that out.

Let me know if this change is better, or worse!  Cheers, jz